src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/remote.py

```diff
@@ -174,6 +186,16 @@ def sendfile(self, token, local_path, remote_path):
                     break
             time.sleep(0.5)
 
+    def get_os_version(self):
```

Comment:
Its probably better if we rename this method to avoid confusion? `get_remote_version` or something?

Reply:
100% agree. I've fixed.